### PR TITLE
Update documentation

### LINKED ISSUE #24: Please use `lamin connect` instead of the legacy `lamin load` in next refactor
https://github.com/search?q=repo%3Alaminlabs%2Flaminr%20%22lamin%20load%22&type=code

### PR DESCRIPTION
**Pull request by rcannood**
Fixes #24

# Changes

## Minor changes

* Replace all mentions of `lamin load` with `lamin connect`.

* Improve the `README` (PR #29).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,72 +22,104 @@ Install the development version from GitHub:
...
 
-> [!TIP]
->
-> You can get your token from the [LaminDB web
-> interface](https://lamin.ai/settings).
+## Connect to an instance
+
+To connect to a LaminDB instance, you will first need to run
+`lamin login` OR `lamin load <instance>` in the terminal. This will
+create a `.lamin` file in your home directory with the necessary
```
Comment (lazappi) [inline README.md:35]: I think this is a directory?

File: README.md [hunk 2]
```diff
@@ -22,72 +22,104 @@ Install the development version from GitHub:
...
+credentials.
 
-## Quick start
+``` bash
+lamin login
+lamin load laminlabs/cellxgene
```
Comment (lazappi) [inline README.md:40]: Should it be `connect` here?

File: README.md [hunk 3]
```diff
@@ -22,72 +22,104 @@ Install the development version from GitHub:
...
+```
 
-Let’s first connect to a LaminDB instance:
+Then, you can use the `connect` function to connect to the instance:
```
Comment (lazappi) [inline README.md:43]: Maybe say this is now the R package, not the command line?

**Review by lazappi — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (rcannood) [reply README.md:35]: Indeed!

```suggestion
create a directory in your home directory called `.lamin` with the necessary

File: README.md (hunk 2 above)
Comment (rcannood) [reply README.md:40]: Indeed!

```suggestion
lamin connect laminlabs/cellxgene

**Review by lazappi — APPROVED** [review]

**Pull request merged**